### PR TITLE
Implement i32.const and i64.const

### PR DESCRIPTION
**Pull request by axic**
Depends on #29.

### CODE REVIEW

**Comment by axic** [comment]
@gumb0 @chfast ready for review

File: lib/fizzy/execute.cpp [hunk 1]
```diff
@@ -147,6 +147,18 @@ execution_result execute(Instance& instance, FuncIdx function, std::vector<uint6
             locals[idx] = stack.back();
             break;
         }
+        case Instr::i32_const:
+        {
+            const auto value = read<int32_t>(immediates);
```
Comment (chfast) [inline lib/fizzy/execute.cpp]: Read directly int `uint32_t` type.

File: lib/fizzy/execute.cpp [hunk 2]
```diff
@@ -147,6 +147,18 @@ execution_result execute(Instance& instance, FuncIdx function, std::vector<uint6
...
+            stack.push(static_cast<uint32_t>(value));
+            break;
+        }
+        case Instr::i64_const:
+        {
+            const auto value = read<int64_t>(immediates);
```
Comment (chfast) [inline lib/fizzy/execute.cpp]: Read directly into `uint64_t` type.

File: lib/fizzy/parser_expr.cpp [hunk 3]
```diff
@@ -76,6 +76,22 @@ parser_result<Code> parse_expr(const uint8_t* pos)
             push(code.immediates, imm);
             break;
         }
+
+        case Instr::i32_const:
+        {
+            int32_t imm;
+            std::tie(imm, pos) = leb128s_decode<int32_t>(pos);
```
Comment (chfast) [inline lib/fizzy/parser_expr.cpp]: Convert to `uint32_t` here.

File: lib/fizzy/parser_expr.cpp [hunk 4]
```diff
@@ -76,6 +76,22 @@ parser_result<Code> parse_expr(const uint8_t* pos)
...
+            push(code.immediates, imm);
+            break;
+        }
+
+        case Instr::i64_const:
+        {
+            int64_t imm;
+            std::tie(imm, pos) = leb128s_decode<int64_t>(pos);
```
Comment (chfast) [inline lib/fizzy/parser_expr.cpp]: covert to `uint64_t` here.

**Pull request merged**